src/runtime_src/doc/toc/fpga_device_ready.rst

```diff
@@ -0,0 +1,54 @@
+.. _fpga_device_ready.rst:
+
+
+FPGA device readiness within user VM
+************************************
+
+To make the FPGA in user VM ready to use, the mgmt side has to be ready first. Beside that, depending on how
+the FPGA is deployed, there are still other factors to be considered.
+
+With MSD/MPD or Plugins used
+============================
+
+In this case, :ref:`mailbox.main.rst` has to be opened from user, and  MPD service must be active.
+
+For MSD/MPD, the MSD service must be active also either from hypervisor or mgmt service domain (VM)
+
+For Plugins, the vendor specific plugin package has to be installed
+
+Without MSD/MPD or Plugins used
+===============================
+  
+In this case, :ref:`mailbox.main.rst` is still being used, but there is no open to the mailbox subdevice from user,
+and MPD service is inactive.
+To make the FPGA device ready, the XCL_MAILBOX_REQ_LOAD_XCLBIN opcode has to be disabled from xclmgmt like following, 
+ 
+.. code-block:: bash
+
+        # In host 
+        Host>$ sudo xbmgmt dump --config --output /tmp/config.ini -d bdf
+
+        # Edit the dumped ini file and change the value to key 'mailbox_channel_disable' 
+        # mailbox_channel_disable=0x100,
+        # where 0x100 is 1 << XCL_MAILBOX_REQ_LOAD_XCLBIN, as defined as below
+        # XCL_MAILBOX_REQ_UNKNOWN =		0,
+        # XCL_MAILBOX_REQ_TEST_READY =		1,
+        # XCL_MAILBOX_REQ_TEST_READ =		2,
+        # XCL_MAILBOX_REQ_LOCK_BITSTREAM =	3,
+        # XCL_MAILBOX_REQ_UNLOCK_BITSTREAM =	4,
+        # XCL_MAILBOX_REQ_HOT_RESET =		5,
+        # XCL_MAILBOX_REQ_FIREWALL =		6,
+        # XCL_MAILBOX_REQ_LOAD_XCLBIN_KADDR =	7,
+        # XCL_MAILBOX_REQ_LOAD_XCLBIN =		8,
```

Comment:
Align 8 with the rest of the entries in the column

Reply:
changed